src/GoTrueClient.ts

```diff
@@ -13,8 +13,11 @@ const DEFAULT_OPTIONS = {
 }
 export default class GoTrueClient {
   api: GoTrueApi
```

Comment:
@kiwicopple are these all meant to be public? E.g. should I use `currentUser` instead of the `user` method?

Reply:
Actually I had that there because of the original supabase client.

I think it might be better to make `currentUser` private, then make calls to `user()` yeah? It future-proofs the library in case we need to add enrichments or handle regression later on